rep-I0004.rst

```diff
@@ -295,8 +295,10 @@ Motoman
       2009  ROS_MSG_MOTO_WRITE_IO_GROUP           Write whole group (byte) of 'Network Inputs'
       2010  ROS_MSG_MOTO_WRITE_IO_GROUP_REPLY     -
       2011  ROS_MSG_MOTO_IOCTRL_REPLY             -
+      2012  ROS_MSG_MOTO_READ_MREGISTER           Read M-Register (M000 through M999)
+      2013  ROS_MSG_MOTO_WRITE_MREGISTER          Write M-Register (M000 through M599)
```

Comment:
I haven't checked the docs, but is it correct that only up to `M599` can be written to?

Reply:
This is correct